src/System.Diagnostics.Tracing.Telemetry/tests/System.Diagnostics.Tracing.Telemetry.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <RootNamespace>System.Diagnostics.Tracing.Telemetry.Tests</RootNamespace>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <NoWarn>0618</NoWarn>
```

Comment:
This should be moved out to the general group rather than just being for debug. The CI is failing the release leg because of this.